aten/src/ATen/quantized/Quantizer.cpp

```diff
@@ -487,16 +525,16 @@ inline Tensor new_qtensor_cpu(
   AT_ASSERT(options.device().is_cpu());
 
   native::check_size_nonnegative(sizes);
-  auto* allocator = at::getCPUAllocator();
+  static QAllocator allocator;
```

Comment:
is this allocator only used for qnnpack? I think you'll need to make sure the fbgemm will use the original CPU allocator?

Reply:
It should work for FBGEMM as well at the cost of wasting 8 bytes per allocation.  Do you prefer keeping FBGEMM's code path intact?

Reply:
yeah, I think it makes more sense to keep the original code path, you can use qengine to distinguish between these cases

Reply:
```
if (QNNPACK) {
// use qnnpack allocator
} else {
// use the original allocator
}
```

Reply:
Thanks Jerry.  Will upload a patch with these changes shortly.

Reply:
I agree that we should keep FBGEMM separate. Please use the complier flag as well as `qengine` to use the new allocator for QNNPACK.